Description: Raise elasticsearch version to 1.0.2.Beta2
 Raise elasticsearch version to 1.0.2.Beta2
Author: Andreas Wirooks <andreas.wirooks@1und1.de>

--- logstash-1.3.2.orig/Makefile
+++ logstash-1.3.2/Makefile
@@ -3,7 +3,7 @@
 #   wget or curl
 #
 JRUBY_VERSION=1.7.8
-ELASTICSEARCH_VERSION=0.90.9
+ELASTICSEARCH_VERSION=1.0.0.Beta2
 
 WITH_JRUBY=java -jar $(shell pwd)/$(JRUBY) -S
 JRUBY=vendor/jar/jruby-complete-$(JRUBY_VERSION).jar
